docs/source/concepts/population.rst

```diff
@@ -9,8 +9,77 @@ Population Management
    :local:
    :backlinks: none
 
-Outline
--------
- - Initialization of simulants and the dependency problem.
- - Managing access with population views.
- -
+Since ``Vivarium`` is an agent-based simulation framework, managing a group of
+:term:`simulants` and their attributes is a critical task. Fundamentally, to run
+a simulation we need to be able to create new simulates, update their state
+attributes, and facilitate access to their state so that :term:`components` in
+the simulation can do interesting things based on it. The tooling to support
+working with our simulant population is called the population management system.
+
+The State Table
+---------------
+
+The core representation of simulants and their state information in ``Vivarium``
+is a :class:`pandas.DataFrame <Data Frame>` known as the state table. Under this
+representation rows represent simulants while columns correspond to state
+attributes like age, sex or systolic blood pressure. These columns represent one
+of several important resources within ``Vivarium`` that other components can
+draw on. Each of the actions we need to be able to take correspond to a
+manipulation of this state table. The addition of new simulants is the creation
+of rows, the creation of new state attributes is the creation of columns, and
+the reading and updating of state is reading and updating the dataframe itself.
+
+<<TODO: image of state table w/ expanding rows and columns>>
+
```

Comment:
Talk about simulant creation as the next section.  Mention how this is distinct from the initialization of population attributes so you can put off talking about population views for a second.  Play up the distinction between the initial population creation and what happens during the main loop more.  Remember, audience is engineers, so what's happening to the state table in these two cases is important.  initial creation builds an index out of nothing and adds columns piecemeal.  main loop creation builds out a totally separate table before appending.  Pictures (hand-drawn is fine, or get fancy if you like) are super useful.  Then code examples of getting the simulant creator and how to use it (and when you can).  In particular, that initial population creation is done by the framework itself (with a warning that this behavior is expected to change).

Do this as a separate PR from the state table bit